### DISCUSSION TITLE
How to use lldb to debug?

### DISCUSSION THREAD

**Discussion by shanemcguffin** [OpenRCT2/OpenRCT2#17060] [Q&A] [answered]
I see this outdated guide says to use lldb:
https://github.com/OpenRCT2/OpenRCT2/wiki/Debugging-OpenRCT2-on-macOS

Could someone give me a few commands to get going?

Would I attach it to build/OpenRCT2.app? Or the .exe?

When I try either of those I get "error: invalid process" when I issue commands to lldb.

**Comment by shanemcguffin** [comment, marked as answer]
`lldb build/OpenRTC2.app`
`run`

**Reply by IntelOrca** [reply]
Thanks, feel free to edit the wiki with updated information as well.
